sklearn/gaussian_process/kernels.py

```diff
@@ -1704,18 +1744,15 @@ class PairwiseKernel(Kernel):
     """
 
     def __init__(self, gamma=1.0, gamma_bounds=(1e-5, 1e5), metric="linear",
-                 pairwise_kernels_kwargs=None):
+                 **pairwise_kernels_kwargs):
```

Comment:
are you sure this is a good decision? If so, the docstring needs to be changed to include `**`; but I also suspect this should be proposed separately.